<launch>
  <arg name="machine" default="localhost"/>
  <arg name="use_machine" default="true"/>
  <arg name="output" default="screen"/>

  <machine name="localhost" address="localhost" if="$(arg use_machine)"/>

  <node pkg="butia_recognition" type="face_recog.py" output="$(arg output)" name="face_recognition_node" machine="$(arg machine)">
    <rosparam command="load" file="$(find butia_recognition)/config/face_rec.yaml"/>
  </node>
  <include file="$(find butia_world)/launch/redis_cache_reader.launch" />
  <include file="$(find butia_world)/launch/redis_cache_writer.launch" />
</launch>
